### PR TITLE
[Spotify-Controls] Display notification when track changes

### PR DESCRIPTION
**Pull request by alejgr**
## Description

Allows user to enable notifications for Next Track and Previous Track commands

## Screencast

image
image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by alejgarciarodriguez** [comment]
Do I have to add new command preferences to CHANGELOG.md? I think so. 😳

**Comment by thomaspaulmann** [comment]
Hey thanks for the contribution. How about changing the notifcation to a [HUD](https://developers.raycast.com/api-reference/feedback/hud)? That feels better integrated with the Raycast UI.

**Comment by thomaspaulmann** [comment]
I think it's good practice to add a changelog entry. It's a new feature that we should highlight properly. Otherwise existing users might not know about it.

File: extensions/spotify-controls/package.json [hunk 1]
```diff
@@ -39,14 +39,36 @@
...
       "description": "Skips to the previous track in Spotify.",
-      "mode": "no-view"
+      "mode": "no-view",
+      "preferences": [
+        {
+          "name": "previousTrackNotificationEnabled",
+          "type": "checkbox",
+          "label": "Enabled",
+          "required": false,
+          "title": "Show previous track info on change",
```
Comment (thomaspaulmann) [inline extensions/spotify-controls/package.json:49]:
```suggestion
          "label": "Enabled",
          "required": false,
          "title": "Show previous track info on change",
```
```suggestion
          "label": "Show previous track info on change",
          "required": false,
```

Comment (thomaspaulmann) [inline extensions/spotify-controls/package.json:49]:
```suggestion
          "label": "Enabled",
          "required": false,
          "title": "Show previous track info on change",
```
```suggestion
          "label": "Show previous track info on change",
          "required": false,
```

File: extensions/spotify-controls/package.json [hunk 2]
```diff
@@ -39,14 +39,36 @@
...
       "description": "Skips to the next track in Spotify.",
-      "mode": "no-view"
+      "mode": "no-view",
+      "preferences": [
+        {
+          "name": "nextTrackNotificationEnabled",
+          "type": "checkbox",
+          "label": "Enabled",
+          "required": false,
+          "title": "Show next track info on change",
```
Comment (thomaspaulmann) [inline extensions/spotify-controls/package.json:67]:
```suggestion
          "label": "Show next track info on change",
          "required": false,
```

Comment (thomaspaulmann) [inline extensions/spotify-controls/package.json:67]:
```suggestion
          "label": "Show next track info on change",
          "required": false,
```

**Comment by thomaspaulmann** [comment]
Thanks for the addition! Looking forward to see my next tracks :-)

**Pull request merged**